### PR TITLE
TDD Project

### PR DESCRIPTION
**Pull request by Ney2**
- This is a PR for our TDD Project
1. Write a test case for each method
2. Write the methods in the solver class
3. Check the functionality of each method

### CODE REVIEW

File: spec/fizzbuzz_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    end
+
+    it 'Should return buzz when divisible by 5' do
+      expect(solver.fizzbuzz(20)).to eq('buzz')
+    end
+
+    it 'Should return fizzbuzz when divisible by 3 and 5' do
+      expect(solver.fizzbuzz(30)).to eq('fizzbuzz')
+    end
+  end
```
Comment (TedLivist) [inline spec/fizzbuzz_spec.rb]: - [ ] Add test for a scenario when the number is **indivisible** by 3 and 5. In that case, it should return a string version of that number.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Ney2 and @AngomRanjan,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/fizzbuzz_spec.rb (hunk 1 above)
Comment (Ney2) [reply spec/fizzbuzz_spec.rb]: okay thank you for your feedback

**Review by brytebee — APPROVED** [review]
Hi, @Ney2 & @AngomRanjan👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



##

###  **Optional Suggestions:**
- N/A

##

Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ney2** [comment]
Thank you for your response.

**Pull request merged**